### PR TITLE
try to get the party started

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

**Comment by SimonDanisch** [comment]
Woah, only gui for lorenz fails :D

File: src/testruns.jl [hunk 1]
```diff
@@ -1,6 +1,6 @@
-const makiegallery_dir = dirname(dirname(@__DIR__))
+raconst makiegallery_dir = dirname(dirname(@__DIR__))
```
Comment (asinghvi17) [inline src/testruns.jl:1]:
```suggestion
const makiegallery_dir = dirname(dirname(@__DIR__))

**Comment by asinghvi17** [comment]
🎉

**Comment by asinghvi17** [comment]
http://juliaplots.org/MakieReferenceImages/gallery//algorithm/index.html looks off, though.

**Comment by SimonDanisch** [comment]
Why?

**Comment by SimonDanisch** [comment]
I improved volume rendering, so this should be more correct, if I didin't mess up^^

**Comment by asinghvi17** [comment]
It's the layouting - the Scenes overlap.

**Comment by SimonDanisch** [comment]
Ah... I think that's maybe because I changed the padding for  3d scenes... I'll probably leave it like this for now, since better fitting in the scene will need to get addressed in the MakieLayout merger anyways

**Pull request merged**